functions.js

```diff
@@ -146,7 +146,7 @@ Output:
 */
 
 export function getLastItem(array) {
-    return true;
+    return array[3];
```

Comment:
Using array notation like this will always target the fourth value in the array. You will notice the issue once you pass in an array with more OR less than four values. So if you pass in an array like this into the function --- ['cat', 'dog', 'sheep', 'llama', 'horse'] --- you would return 'llama' instead of the last item 'horse.' You could try targeting the length of the array and - 1 from it to always get the last item.